big_tests/tests/muc_SUITE.erl

```diff
@@ -85,15 +84,37 @@
         items :: [#xmlel{}]
         }).
 
+host_type() ->
+    ct:get_config({hosts, mim, host_type}).
+
 %%--------------------------------------------------------------------
 %% Suite configuration
 %%--------------------------------------------------------------------
 
-all() -> [
-          {group, disco},
-          {group, disco_non_parallel},
-          {group, disco_rsm},
-          {group, disco_rsm_with_offline},
+disco_disabled() ->
+    try
+        ct:get_config({disable_disco_tests}, false)
+    catch _:_ ->
+              false
+    end.
+
+s2s_disabled() ->
+    try
+        ct:get_config({disable_s2s_tests}, false)
+    catch _:_ ->
+              false
+    end.
+
+all() -> 
+    case disco_disabled() of
+        true ->
+            [];
+        false ->
+            [{group, disco},
+             {group, disco_non_parallel},
+             {group, disco_rsm},
+             {group, disco_rsm_with_offline}]
+    end ++ [
```

Comment:
What about doing a trick like Denys did at https://github.com/esl/MongooseIM/blob/5166ab32fbeef8f1e369c2ec3b3e74a8d2114663/big_tests/dynamic_domains.spec#L15 seems clearer, smaller diff, doesn't make a mess of the test-suite, and it's easier to add and remove in the future 🤔

Reply:
done